src/arch/x86_64/bits.zig

```diff
@@ -411,20 +411,17 @@ pub const Memory = union(enum) {
         dword,
         qword,
         tbyte,
+        dqword,
 
         pub fn fromSize(size: u32) PtrSize {
-            return if (size <= 1)
-                .byte
-            else if (size <= 2)
-                .word
-            else if (size <= 4)
-                .dword
-            else if (size <= 8)
-                .qword
-            else if (size == 10)
-                .tbyte
```

Comment:
Is `tbyte` left out on purpose here?

Reply:
Between the abi size of `tbyte` actually being 12 or 16 bytes and the `<=` structure of the other entries, having a `tbyte` case seems to makes this function less useful.  Something like a full `Type` argument to properly distinguish these cases could work.

Reply:
Mhm, that actually might clean up a lot of the code if we start accepting `Type` in places like this. I like this idea 👍